apps/docs/components/react/browser-support.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+---
+
+# Browser support
+
+This documentation provides information about the browser support for Storefront-ui, a powerful and versatile front-end UI framework. Understanding the browser compatibility of a UI framework is crucial for developers to ensure that their applications work seamlessly across different browsers and versions. This document will outline the supported browsers, their versions, and any specific considerations or known issues related to compatibility.
+
+## The following browsers are officially supported by Storefront-ui:
+
+**2 last stable versions of each browser**
```

Comment:
```suggestion
**The 2 latest stable versions of:**
```

I think this better sets up the list, but feel free to ignore.

Reply:
Good suggestion, here's change https://github.com/vuestorefront/storefront-ui/pull/2883/commits/d9f815bdedbf9008a218a932bd4aa3c303c5f653